app/lib/common/widgets/chat/convo_card.dart

```diff
@@ -58,15 +63,76 @@ class _ConvoCardState extends ConsumerState<ConvoCard> {
                 latestMessage: latestMsg,
               )
             : const SizedBox.shrink(),
-        trailing: latestMsg != null
-            ? _TrailingWidget(
-                // controller: receiptController,
-                room: widget.room,
-                latestMessage: latestMsg,
-                activeMembers: activeMembers,
-                userId: client!.userId().toString(),
-              )
-            : const SizedBox.shrink(),
+        trailing: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          children: [
+            latestMsg != null
+                ? _TrailingWidget(
+                    // controller: receiptController,
```

Comment:
You can remove this comment. It doesn't exists neither we're looking forward too.